src/ProcessManager.php

```diff
@@ -902,10 +902,16 @@ protected function newSlaveInstance($port)
         // we can not use -q since this disables basically all header support
         // but since this is necessary at least in Symfony we can not use it.
         // e.g. headers_sent() returns always true, although wrong.
-        $commandline = $this->phpCgiExecutable . ' -C ' . ProcessUtils::escapeArgument($file);
+        if (method_exists('Symfony\Component\Process\ProcessUtils', 'escapeArgument')) {
+            $commandline = 'exec ' . $this->phpCgiExecutable . ' -C ' . ProcessUtils::escapeArgument($file);
+        } else {
+            $commandline = ['exec', $this->phpCgiExecutable, '-C', $file];
+            $process = new \Symfony\Component\Process\Process($commandline);
+            $commandline = $process->getCommandLine();
+        }
```

Comment:
Well, we could simply go the way with an array all the time, can't we? Or does older Symfony not support arrays as first argument for `Process`?

```
$commandline = ['exec', $this->phpCgiExecutable, '-C', $file];
// use exec to omit wrapping shell	         // use exec to omit wrapping shell
$process = new Process('exec ' . $commandline);	+        $process = new Process($commandline);

`